src/main/java/bio/terra/service/filedata/FileMetadataUtils.java

```diff
@@ -77,10 +80,33 @@ public FireStoreDirectoryEntry makeDirectoryEntry(String lookupDirPath) {
   public String makeLookupPath(String fullPath) {
     String temp = StringUtils.prependIfMissing(fullPath, "/");
     temp = StringUtils.removeEnd(temp, "/");
-    return ROOT_DIR_NAME + temp;
+    temp = StringUtils.prependIfMissing(temp, ROOT_DIR_NAME);
+    return temp;
```

Comment:
That's fair. It's just used across a few lookups and by adding this, it allows us to handle both cases: The case where we've already prepended it and not.  If we've already prepended it, then it's just another sanity check that it's in the right format. I don't see the harm in this imprecision, but could be convinced otherwise!